.github/workflows/main.yml

```
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  setup-java-maven:
+    uses: ./.github/workflows/setup-java-maven.yml
```

actionlint (1.7.4)

error while parsing reusable workflow "./.github/workflows/setup-java-maven.yml": "on:" is not found

(expression)